### ISSUE TITLE
flushing doesnt seem to happen automatically even after 5-10 minutes

### ISSUE THREAD

**Issue by pathsny** [pgte/alfred#3]
I'm trying to use alfred for a persistent server, so I never actually close the database. But the database file never grows beyond zero. Here's my code

var Alfred = require('alfred');
Alfred.open('./ircdb', function(err, db){
    if (err) {throw err; }

```
db.on('error', function(err) {
    console.log('OMG, an error occurred!: ' + err.message);
});
console.log("in before flynn");

var Nick = db.define('Nick');
Nick.property('nick', 'string');
Nick.property('nick_id', Number);
Nick.index('nick', function(nick){
    return nick.nick;
});
Nick.index('nick_id', function(nick){
    return nick.nick_id;
});

Nick.find({nick: {$eq: "thenu"}}).all(function(value){
    console.log("from find");
    console.log(arguments);  
})


var nn = Nick.new({nick: "thenu", nick_id: Math.floor(Math.random()*1000)});
nn.save(function(err){
    console.log(err);
    console.log(nn);
})
```

})

**Comment by pathsny** [comment]
fixed here
https://github.com/pathsny/alfred

**Comment by pgte** [comment]
Fixed on new release 0.5.3
